src/graph/UpdateEdgeExecutor.cpp

```diff
@@ -190,68 +190,36 @@ void UpdateEdgeExecutor::finishExecution(storage::cpp2::UpdateResponse &&rpcResp
         rows.back().set_columns(std::move(row));
     }
     resp_->set_rows(std::move(rows));
-    doFinish(Executor::ProcessControl::kNext);
 }
 
-
 void UpdateEdgeExecutor::setupResponse(cpp2::ExecutionResponse &resp) {
     if (resp_ == nullptr) {
         resp_ = std::make_unique<cpp2::ExecutionResponse>();
     }
     resp = std::move(*resp_);
 }
 
-
-void UpdateEdgeExecutor::insertReverselyEdge(storage::cpp2::UpdateResponse &&rpcResp) {
-    std::vector<storage::cpp2::Edge> edges;
-    storage::cpp2::Edge reverselyEdge;
-    reverselyEdge.key.set_src(edge_.dst);
-    reverselyEdge.key.set_dst(edge_.src);
-    reverselyEdge.key.set_ranking(edge_.ranking);
```

Comment:
why not set ranking

Reply:
The edge_ is reused here.  The ranking has been set in the previous step.